packages/core/components/VisxShape.tsx

```diff
@@ -21,23 +17,41 @@ interface VisxShapeProps {
   left: number
   top: number
   display: boolean
-  index: GlyphKeys
+  index: number
   stroke: string
   key: string | number
+  strokeWidth?: number
 }
 
-const VisxShape: React.FC<VisxShapeProps> = ({ fill, size, left, top, display, index, stroke, key }) => {
+const VisxShape: React.FC<VisxShapeProps> = ({
+  fill,
+  size,
+  left,
+  top,
+  display,
+  index,
+  stroke,
+  key,
+  strokeWidth = 0
+}) => {
   if (!display) {
-    return null
+    return <g></g>
   }
 
   const Shape = glyphs[index]
```

Comment:
I think instead of using an index here the shape name should be used.

Reply:
Because we use indexes of series key. using names would add more complexity.